spec.html

```diff
@@ -37039,11 +37041,8 @@ <h1>The %AsyncFromSyncIteratorPrototype% Object</h1>
           <h1>%AsyncFromSyncIteratorPrototype%.next ( _value_ )</h1>
           <emu-alg>
             1. Let _O_ be the *this* value.
+            1. Assert: Type(_O_) is Object and _O_ has a [[SyncIteratorRecord]] internal slot.
```

Comment:
shouldn't this be `RequireInternalSlot`

Reply:
good catch; want to submit a followup?

Reply:
actually no, this is an assert, while RequireInternalSlot throws.